coap-gateway/coapconv/coapconv.go

```diff
@@ -161,12 +161,12 @@ func MakeCommandMetadata(sequenceNumber uint64, connectionID string) pbCQRS.Comm
 	}
 }
 
-func MakeConfirmResourceRetrieveRequest(deviceID, href, correlationId string, authCtx pbCQRS.AuthorizationContext, connectionID string, req *pool.Message) pbRA.ConfirmResourceRetrieveRequest {
+func MakeConfirmResourceRetrieveRequest(deviceID, href, correlationId string, authCtx *pbCQRS.AuthorizationContext, connectionID string, req *pool.Message) pbRA.ConfirmResourceRetrieveRequest {
```

Comment:
We're currently working only with devices, but the plan is to host resources also by other services. In such a scenario, resources are published by service and not by the device. What do you think about making this transparent in our naming? Renaming everywhere deviceId to e.g. `resourceServerID `? I wouldn't do it in this MR because it will blow up, but I would do it in v2.

Reply:
too long name but I'm not against it.